Fix build failure with +smyrna variant:

main.c:236:19: error: implicit declaration of function 'readln' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://gitlab.com/graphviz/graphviz/-/issues/2631
https://gitlab.com/graphviz/graphviz/-/commit/b215dbe2cbcfa3e76e242a444cf69aac54a7a046
--- cmd/smyrna/main.c.orig
+++ cmd/smyrna/main.c
@@ -178,7 +178,7 @@ static void windowedMode(int argc, char *argv[]) {
     gtk_main();
 }
 
-#if !defined(__APPLE__) && !defined(_WIN32)
+#ifndef _WIN32
 /// `readlink`-alike but dynamically allocates
 static char *readln(const char *path) {
 
